docs/UserGuide.md

```diff
@@ -538,22 +548,32 @@ Precise outputs when the command fails:
 
 ### Sorting sellers: `ssort`
 
-Sorts the sellers' list by the provided attribute and in the given direction.
+Sorts the sellers' list either by the list's default order, or by a provided prefix and direction.
```

Comment:
Mention explicitly that leaving it blank sorts by default order (not super clear here.)